tools/querytee/proxy_backend_test.go

```diff
@@ -84,7 +84,7 @@ func Test_ProxyBackend_createBackendRequest_HTTPBasicAuthentication(t *testing.T
 				orig.Header.Set("X-Scope-OrgID", testData.clientTenant)
 			}
 
-			b := NewProxyBackend("test", u, time.Second, false, false)
+			b := NewProxyBackend("test", u, time.Second, false, false, BackendConfig{})
```

Comment:
[nit] Do you think we should have a `defaultBackendConfig()` instead (we can set it to return `BackendConfig{}` for now) or is it unlikely that we'll ever need to set some values as default?